src/languages/en.js

```diff
@@ -146,6 +146,14 @@ export default {
         youCanAlso: 'You can also',
         openLinkInBrowser: 'open this link in your browser',
     },
+    magicCodeModal: {
+        succesfulSignInTitle: 'Abracadabra,\nyou are signed in!',
+        succesfulSignInDescription: 'Head back to your original tab to continue.',
+        title: 'Here is your magic code',
+        description: 'Enter it to sign in',
```

Comment:
For this QA flow, it doesn't really make sense to have `Enter it to sign in` because where would the user be entering the code? 

> 6. Now that you are signed in, enter that link again in a new tab
> 7. Verify that you see a full page with the magic code
> i<img width="400" alt="" src="https://user-images.githubusercontent.com/16747822/217982575-b833560b-bb8c-4d0e-9296-a479b2cb838d.png">

Reply:
@shawnborton Could you please help me with a full copy here?

Reply:
Let's tag @Expensify/marketing for help on this one. I think maybe you can create a GH issue over in Expensify/Expensify and add the marketing-copy label to it and it will get auto-assigned?

If I had to take a stab though, I might say `Please enter the code using the device where it was originally requested.`

Reply:
I will use your suggestion for now and then I will create an issue as you said to review all the copies in these screens. Thank you @shawnborton !